### DISCUSSION TITLE
Disable device tracker feature?

### DISCUSSION THREAD

**Discussion by tismofied** [Vaskivskyi/ha-asusrouter#342] [Q&A] [answered]
I am using automation to unlock door when home. It works perfect with HA tracking my phone but when I install this integration it messes with my status "home" "not_home" and the door lock keeps unlocking certain times when the integration detects phone as being home even though I never left the house. 
Anyway to disable phone device tracker and still be able to use this wonderful integration?

**Comment by Vaskivskyi** [comment]
Hey,

AsusRouter only created its own device trackers and does not influence any other entities.

Are you using a `person` entity for triggering automation? Then don't append AsusRouter device_tracker to the person profile.

In any case:
- device trackers from AsusRouter can be disabled altogether (using the integration options)
- you can disable a single device_tracker entity as any other entity

Maybe if you post here your automation trigger I would be able to help you more

**Comment by tismofied** [comment]
Hey,

I just double checked on my automation and I am indeed using iPhones to track our entering and leaving Home zone.
I am still new to device tracking in HA, so I don't really know the difference between my iPhone tracking and my router.
Hopefully one of the youtubers could feature your integration and show how it is supposed to be used. I've been following your integration for a bit now and it matured quite a bit with awesome features. I just don't know how to use them yet.

I have recently switched phones so maybe installing HA companion in a different phone reset the automation trigger and your ASUS integration picked it up and started reporting my zone status? not really sure lol.

**Comment by Vaskivskyi** [comment, marked as answer]
Device trackers from phones and from routers are of different types.

### Phone
This device tracker is a `TrackerEntity` as per HA docs. It is based on GPS. The phone knows its location and sends it to HA. As a result, `home` / `not_home` is based on the coordinates and whether they are inside the `home` zone.

<details>
<summary>Example - TrackerEntity</summary>

![image](https://user-images.githubusercontent.com/6554415/202845106-9bd061ef-28ff-45ec-a36c-a65f48b60e46.png)
</details>

### Router
This is a `ScannerEntity`. The router only knows whether a device is connected to it. So if it is connected - the state is `home`, if not - `not_home`. Obviously, if WiFi connects/disconnects, it will be switching back and forth.

<details>
<summary>Example - ScannerEntity</summary>

![image](https://user-images.githubusercontent.com/6554415/202845010-cd8eaa0f-a6a4-4cac-85e1-6a5c9aff4379.png)
</details>

So just make sure, the entity based on the GPS is used for automation. That would be the one from the Companion app
